docs/DeveloperGuide.md

```diff
@@ -944,6 +945,23 @@ Use case ends.
 
         Use case resumes from step 3.
 
+### UC-023 Delete an alias
+
+**MSS**
+
+1. User wants to delete an alias.
```

Comment:
Maybe change `wants` to `requests`. `Wants` doesn't indicate that the user is performing an action on SunRez.

The use case above (UC-022) has the same thing.